discourse-mixin/panels.libsonnet

```diff
@@ -0,0 +1,148 @@
+local g = import './g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+
+{
+  new(this): {
+    local signals = this.signals,
+
+    // Overview dashboard panels
+    trafficByResponseCode:
+      commonlib.panels.network.timeSeries.traffic.new(
+        'Traffic by response code',
+        targets=[signals.overview.httpRequests.asTarget()]
+      )
+      + g.panel.timeSeries.options.legend.withAsTable(true)
+      + g.panel.timeSeries.options.legend.withPlacement('right')
+      + g.panel.timeSeries.panelOptions.withDescription('Rate of HTTP traffic over time for the entire application. Grouped by response code.')
+      + g.panel.timeSeries.standardOptions.withUnit('reqps'),
+
+    activeRequests:
+      commonlib.panels.network.timeSeries.traffic.new(
+        'Active requests',
+        targets=[signals.overview.activeRequests.asTarget()]
+      )
+      + g.panel.timeSeries.panelOptions.withDescription('Active web requests for the entire application')
+      + g.panel.timeSeries.standardOptions.withUnit('reqps'),
+
+    queuedRequests:
+      commonlib.panels.network.timeSeries.traffic.new(
+        'Queued requests',
+        targets=[signals.overview.queuedRequests.asTarget()]
+      )
+      + g.panel.timeSeries.panelOptions.withDescription('Queued web requests for the entire application.')
+      + g.panel.timeSeries.standardOptions.withUnit('reqps'),
+
+    pageViews:
+      commonlib.panels.network.timeSeries.traffic.new(
+        'Page views',
+        targets=[signals.overview.pageViews.asTarget()]
+      )
+      + g.panel.timeSeries.panelOptions.withDescription('Rate of pageviews for the entire application. Grouped by type and service.')
+      + g.panel.timeSeries.standardOptions.withUnit('none'),
```

Comment:
Do we want to change the unit to something more semantic like views/sec?

Reply:
> [panel-units-rule] 'Discourse overview': Dashboard 'Discourse overview', panel 'Page views' has no or invalid units defined: 'views/sec'

It fails linting :/

Reply:
My bad, I was asking if there is something as meaningful as views/sec, not specifically that unit.